tools/webpack/script-modules.js

```diff
@@ -89,11 +89,13 @@ module.exports = {
 	},
 	output: {
 		devtoolNamespace: 'wp',
-		filename: './build-module/[name].min.js',
+		filename: `[name]${
+			baseConfig.mode === 'production' ? '.min' : ''
```

Comment:
I need to verify that later with WP packages that land in the `build` folder. I have an impression that a custom webpack plugin takes care of adding the second file based on some strategy. It’s complicated and every time I need to research it. Context: https://github.com/WordPress/gutenberg/tree/trunk/packages/readable-js-assets-webpack-plugin

The gist of it is. It might be fine as is or we need to use the same file extension.

Reply:
Good spot, I was wondering about this. It should always just write `.min.js` like packages build does:

https://github.com/WordPress/gutenberg/blob/1bbe4c4936a7961d5482e826d5d59aa996906e17/tools/webpack/packages.js#L133

I'll update it.

Reply:
With `.min.js`:

- `rm -fr build-module`
- `NODE_ENV=production npm run build`
- `ls -lah build-module/interactivity`

```
  256 Sep 20 11:42 ./
  224 Sep 20 11:42 ../
 108K Sep 20 11:42 debug.js
  45K Sep 20 11:42 debug.min.js
 291K Sep 20 11:42 debug.min.js.map
  97K Sep 20 11:42 index.js
  36K Sep 20 11:42 index.min.js
 257K Sep 20 11:42 index.min.js.map
```